book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- Kindly Getters and Setters for `rentals` should be present with attr_accessor, as we intend to both access (get) as well as write (set) it. please ensure to improve upon this aspect. 👍

Reply:
@binhussen Getters and setters have been added for the rentals attribute using attr_accessor. Now, you can both access (get) and write (set) the rentals attribute. Thank you for your suggestion.